src/Core/src/VisualDiagnostics/BootstrapHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+#pragma warning disable CS8600
+namespace Microsoft.Maui
```

Comment:
Is it worth _linking_ the file (instead of copying it) and doing an `#if` to change the namespace?